isic/ingest/services/publish/__init__.py

```diff
@@ -94,7 +101,62 @@ def cohort_publish(*, publish_request: PublishRequest) -> None:
     sync_elasticsearch_indices_task.delay_on_commit()
 
 
+def embed_iptc_metadata(blob: FieldFile, attribution: str, license_: str) -> Path:
```

Comment:
This function should be a context manager, so the temp file is deleted automatically if an exception occurs.

Reply:
If this yields a readable IO object instead of a Path, it'll be cleaner to use and easier to replace / mock.